samcli/local/apigw/local_apigw_service.py

```diff
@@ -1,4 +1,8 @@
 """API Gateway Local Service"""
+# pylint: disable=too-many-lines
+# TODO (lucashuy): this module needs to be refactored
```

Comment:
This module is now too large for our linter (just a tad over 1000 lines). This will need to be refactored in a future PR since this one is already a bit loaded.